### ISSUE TITLE
Nothing is displayed in dashboard

### ISSUE THREAD

**Issue by ErlanBazarov** [taskforcesh/issues#94]
![image](https://user-images.githubusercontent.com/56782170/172586250-4b0709fe-e4aa-438c-8c59-8f8330b1e3b4.png)
I logged in with 3 different accounts and faced the same problem

**Comment by manast** [comment]
Is it possible you happen to login during a deployment? Do you still have this issue?

**Comment by ErlanBazarov** [comment]
Yeah, I still have this issue

**Comment by ErlanBazarov** [comment]
I am not seeing this issue if I use a different browser than chrome or a mobile phone. I cleared local storage and cookies in chrome, but problem is still there

**Comment by manast** [comment]
That's very strange, can you check the console in chrome to see if there is something printed there that can give us a hint on what is going on?

**Comment by ErlanBazarov** [comment]
![image](https://user-images.githubusercontent.com/56782170/172759718-13e8aa32-f154-4cd8-a7bd-397d4fb66e7c.png)
In this url https://taskforce.sh/connections

**Comment by manast** [comment]
What about the network tab? is there any request that is failing?

**Comment by ErlanBazarov** [comment]
No, no request is failing in network tab

**Comment by manast** [comment]
Did you try in a different computer? Because this is the only report we got with this issue and I am not able to reproduce it either.

**Comment by ErlanBazarov** [comment]
Yes, it is working on a different computer. It is even working on a different chrome profile that doesn't have any cache.  
I am assuming it is problem with a cache, but I tried to hard reload, clear local storage, cookies, and whatever can be cleared from the application tab. I also cleared taskforce, stripe, app.chaport from _Privacy and security > Site Settings > Cookies & Site Data > See all cookies and site data._
I restarted my laptop as well
